src/reversible_tokenize.jl

```diff
@@ -0,0 +1,89 @@
+# A simple reversible tokenizer
+"""
+A simple reversible tokenizer
+
+```
+tokenized = rev_tokenizer(instring)
+de_tokenized = rev_detokenizer(token)
+```
+
+The rev_tokenize tokenizer splits into token based on space, punctuations and special symbols and in 
+addition it leaves some MERGESYMBOLS for the tokens to be re-arranged when needed 
+using the rev_detokenize.
+It uses a character based approach for splitting and re-merging.
+
+Parameters:
+
+- instring		= Input string to be tokenized 
+- token			= Collection to tokens i.e String Array
+
+"""
+const global MERGESYMBOL = '\ue302'
```

Comment:
```suggestion
const MERGESYMBOL = '\ue302'
```

`global` does nothing when called at global scope

Reply:
-Updated